diff --git a/maufbapi/mqtt/conn.py b/maufbapi/mqtt/conn.py
index b688c76..4114863 100644
--- a/maufbapi/mqtt/conn.py
+++ b/maufbapi/mqtt/conn.py
@@ -397,6 +397,9 @@ class AndroidMQTT:
             self.seq_id_update_callback(self.seq_id)
 
     def _on_message_sync(self, payload: bytes) -> None:
+        from ..thrift import ThriftReader
+
+        ThriftReader(payload).pretty_print()
         try:
             parsed = MessageSyncPayload.from_thrift(payload)
         except Exception:
